src/ensembl/production/metadata/updater/core.py

```diff
@@ -574,7 +566,7 @@ def get_or_new_genebuild(self, species_id, meta_session, source=None, existing=F
                 dataset_uuid=str(uuid.uuid4()),
                 dataset_type=dataset_type,
                 name="genebuild",
-                version=genebuild_version,
+                version=dataset_version,
```

Comment:
I very much disagree on this. I strongly believe that we should have version as a required key and to have it change on each release. This is a very minor thing to ask the genebuild teams and would keep our code simple and our workflows simple and easy to pick up. We can easily require that the dates change as well, but I think we should limit the code to have one big decider on updating, for genesets, otherwise I predict a mountain of bugs in the future that will be difficult to debug.

Reply:
If the date don't change, the output FTP will be the same, then overriding the previous data available and probably download by someone. Breaking one core thing about reproducibility. We could keep the `dataset.version` attribute though. from the version, meaning every time you want to get the genebuild_version you'll have to scroll all the dataset attributes to fetch the one you want.